ui/app/components/http-requests-table.js

```diff
@@ -0,0 +1,51 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+import { assign } from '@ember/polyfills';
+
+/**
+ * @module HttpRequestsTable
+ * HttpRequestsTable components render a table with the total number of HTTP Requests to a Vault server per month.
+ *
+ * @example
+ * ```js
+ * const COUNTERS = [
+ *    {
+ *       "start_time": "2019-05-01T00:00:00Z",
+ *       "total": 50
+ *     }
+ * ]
+ *
+ * <HttpRequestsTable @counters={{COUNTERS}} />
+ * ```
+ *
+ * @param counters {Array} - A list of objects containing the total number of HTTP Requests for each month. `counters` should be the response from the `/internal/counters/requests` endpoint.
+ */
+
+export default Component.extend({
+  classNames: ['http-requests-table'],
```

Comment:
I've been trying to move as many components as possible to `tagName: ''` and then move any `classNames` to a wrapping element in the template since that's going to be more look more like what we're doing in the future with Glimmer components. What do you think about doing things that way?

Reply:
oh sure, that sounds great. thanks for the tip!